hvpy/facade.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def _copy_docstring(input_class):
+    def decorator(func):
+        input_class_docstr = input_class.__doc__.split("\n")[1:]
+        func_docstr = func.__doc__.split("\n")[1:]
+        input_class_docstr[0] = func_docstr[0]
+        input_class_docstr = "\n".join(input_class_docstr)
+        func.__doc__ = input_class_docstr
+        return func
+
+    return decorator
```

Comment:
```suggestion
    def decorator(func):
        func.__doc__ = func.__doc__ + "\n".join(input_class.__doc__.split("\n")[1:])
        return func
    return decorator
```
I've not tested this.

Reply:
I will try to simplify this.